lib/std/os/linux.zig

```diff
@@ -69,7 +70,10 @@ pub const LOCK = arch_bits.LOCK;
 pub const MMAP2_UNIT = arch_bits.MMAP2_UNIT;
 pub const REG = arch_bits.REG;
 pub const SC = arch_bits.SC;
-pub const Stat = arch_bits.Stat;
+pub const Stat = if (native_arch == .riscv32)
+    @compileError("No stat structure on this architecture.")
```

Comment:
Just noting that this isn't strictly speaking true. It is there in the UAPI headers. It's just completely useless because no syscalls use it. So having it defined is more of a portability trap than emitting a compile error.